jumanji/specs_test.py

```diff
@@ -648,7 +612,9 @@ def test_bounded_array(self) -> None:
     def test_discrete_array(self) -> None:
```

Comment:
NIT: some tests have docstring descriptions and some don't. I'll leave it up to you if they are all necessary

Reply:
Overall, I try to improve as I update the code. If I see tests that are unclear and do not have a docstring I will add one.